app/frontend/src/pages/chat/Chat.tsx

```diff
@@ -607,6 +615,30 @@ const Chat = () => {
                         )}
                     />
 
+                    <Dropdown
```

Comment:
I wish that I had refactored these settings to be in a single component, as now we have duplicated code each time we add a new setting. This is fine for this PR, but eventually I want to try a single Settings.tsx across Chat/Ask.

Reply:
Agreed, refactoring to a single file is a good future improvement (considered during the RAG HACK project). Let's proceed with this PR for now then.